src/jti_background.js

```diff
@@ -44,7 +44,8 @@ if (browserType !== 'Firefox') {
 // -------------------------------------------------------------------------- //
 
 var StorageID = 'Jira-Template-Injector';
-var emptyData = {'options': {'limit': []}, 'templates': {}};
+var rateStatus = 'JTI-Rate-Status';
```

Comment:
I don't think we should be using another ID. We should be able to store and fetch all the data under the one StorageID